_includes/footer.html

```diff
@@ -1,53 +1,12 @@
 <footer class="global-footer">
   <div class="container">
-    <div class="row">
-      <div class="col-6 col-md-3">
-        <h6>Developers</h6>
-        <ul class="list-unstyled">
-          <li><a href="https://en.opensuse.org/Portal:Development">Documentation</a></li>
-          <li><a href="https://build.opensuse.org/">Build service</a></li>
-          <li><a href="https://bugzilla.opensuse.org/">Bugzilla</a></li>
-          <li><a href="https://github.com/openSUSE">Github</a></li>
-          <li><a href="https://features.opensuse.org/">openFATE</a></li>
-        </ul>
-      </div>
-      <div class="col-6 col-md-3">
-        <h6>Information</h6>
-        <ul class="list-unstyled">
-          <li><a href="https://news.opensuse.org/">News</a></li>
-          <li><a href="https://doc.opensuse.org/release-notes/">Release notes</a></li>
-          <li><a href="https://events.opensuse.org/">Events</a></li>
-          <li><a href="http://planet.opensuse.org/">Planet</a></li>
-          <li><a href="https://shop.opensuse.org/">Shop</a></li>
-        </ul>
-      </div>
-      <div class="col-6 col-md-3">
-        <h6>Community</h6>
-        <ul class="list-unstyled">
-          <li><a href="https://forums.opensuse.org/">Forums</a></li>
-          <li><a href="https://connect.opensuse.org/">Connect</a></li>
-          <li><a href="https://www.facebook.com/groups/opensuseproject/">Facebook group</a></li>
-          <li><a href="https://plus.google.com/communities/115444043324891769569">Google+ group</a></li>
-          <li><a href="https://en.opensuse.org/openSUSE:Mailing_lists_subscription">Mail lists</a></li>
-          <li><a href="https://en.opensuse.org/openSUSE:IRC_list">IRC channels</a></li>
-        </ul>
-      </div>
-      <div class="col-6 col-md-3">
-        <h6>Social Media</h6>
-        <ul class="list-unstyled">
-          <li><a href="https://www.facebook.com/en.openSUSE">Facebook</a></li>
-          <li><a href="https://plus.google.com/+openSUSE">Google+</a></li>
-          <li><a href="https://twitter.com/opensuse">Twitter</a></li>
-          <li><a href="https://www.youtube.com/user/opensusetv">YouTube</a></li>
-          <li><a href="https://t.me/opensusenews">Telegram</a></li>
-        </ul>
-      </div>
-    </div>
-
-    <div class="footer-copyright-language-row">
+    <div class="d-flex justify-content-between">
       <div class="footer-copyright">
-        © 2011–2017 openSUSE contributors
+        © 2011–{{ site.time | date: '%Y' }} openSUSE contributors
```

Comment:
We should probably not have a starting date for the theme, since we don't know when the sites that will use the theme started out